src/widget/wcuemenupopup.h

```diff
@@ -2,14 +2,34 @@
 
 #include <QLabel>
 #include <QLineEdit>
+#include <QMouseEvent>
 #include <QPushButton>
 
+#include "control/pollingcontrolproxy.h"
 #include "preferences/colorpalettesettings.h"
 #include "track/cue.h"
 #include "track/track_decl.h"
+#include "util/parented_ptr.h"
```

Comment:
This is not used.
Did you intend to use it for `m_pSavedLoopCue`?
Could as well use it for the othe rmembers `m_pEditLabel`, `m_pColorPicker`, `m_pDeleteCue`.

Reply:
I meant to use `unique_ptr` - updated now